### DISCUSSION TITLE
Weather commands?

### DISCUSSION THREAD

**Discussion by soulsavyleo** [Pryaxis/TShock#1840] [Q&A]
I don't know if this as been asked already, but I'm new to the plugin, so how do I make it rain in my server? I do remember there being some command like /rain before but I can't find it in the help list. Anything I can do?

**Comment by hakusaro** [comment]
I think you're looking for `/worldevent rain` (you can see this on [the changelog](https://github.com/Pryaxis/TShock/blob/general-devel/CHANGELOG.md#tshock-440-pre-release-3)).
